readthedocs/builds/managers.py

```diff
@@ -85,6 +86,41 @@ def get_object_or_log(self, **kwargs):
             log.warning('Version not found for given kwargs. %s' % kwargs)
 
 
+class InternalVersionManagerBase(VersionManagerBase):
+
+    """
+    Version manager that only includes internal version.
+
+    It will exclude PULL_REQUEST type from the queries
+    and only include BRANCH, TAG, UNKONWN type Versions.
+    """
+
+    def get_queryset(self):
+        return super().get_queryset().exclude(type=PULL_REQUEST)
+
+
+class ExternalVersionManagerBase(VersionManagerBase):
+
+    """
+    Version manager that only includes external version.
+
+    It will only include PULL_REQUEST type Versions in the queries.
+    """
+
+    def get_queryset(self):
+        return super().get_queryset().filter(type=PULL_REQUEST)
+
+
 class VersionManager(SettingsOverrideObject):
     _default_class = VersionManagerBase
     _override_setting = 'VERSION_MANAGER'
+
+
+class InternalVersionManager(SettingsOverrideObject):
+    _default_class = InternalVersionManagerBase
+    _override_setting = 'INTERNAL_VERSION_MANAGER'
```

Comment:
Let's remove these `_override_setting` variable from all these managers. We have another way to override them explicitly now.

Reply:
Done!